torch/csrc/jit/import.cpp

```diff
@@ -196,8 +210,13 @@ at::Tensor ScriptModuleDeserializer::loadTensor(
 }
 
 void ScriptModuleDeserializer::convertModule(
-    const torch::ModuleDef& module_def) {
+    const torch::ModuleDef& module_def,
+    const string& model_metadata,
+    bool main_module) {
   std::shared_ptr<script::Module> module = moduleLookup_(moduleStack_);
+  if (main_module) {
```

Comment:
Boolean flag is weird here. Probably because it is mixing up the purpose of a Module and a Model.